### PR TITLE
Expanded ReadMe

### PR DESCRIPTION
**Pull request by ivan-egorov42**
@yegor256 please, have a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,34 @@ More information by:
 
 If you want to change colors in vim as you wish, you can see the list of highlight groups [here](https://github.com/objectionary/eo-vim/blob/master/syntax/README.md).
 
+## Highlight setting
```
Comment (yegor256) [inline README.md]: @ivan-egorov42 we already have "Configuration" title. "Highlight setting" doesn't look like a good fit for the same level heading

**Comment by yegor256** [comment]
@ivan-egorov42 see above

**Comment by yegor256** [comment]
@ivan-egorov42 also, I think we should delete the second README, to avoid information duplication

**Comment by ivan-egorov42** [comment]
@yegor256 please, have a look.

**Comment by yegor256** [comment]
@ivan-egorov42 thanks!

**Pull request merged**